src/components/Menu/Menu.jsx

```diff
@@ -1,27 +1,25 @@
 import React from 'react';
 import { MenuCategory } from '..';
 
-export class Menu extends React.Component {
-  render() {
-    const { categories, breads, addToOrder } = this.props;
+export const Menu = ({
+  categories,
+  breads,
+  addToOrder
+}) => (
+  <div className="menu">
+    <h1>Menu</h1>
+    {categories.map((category, i) => {
```

Comment:
If you destructure `title` and `items` within the params you can implicitly return `MenuCategory` to tidy things up a bit 👍 

```
{categories.map(({ title, items }, i) => (
  <MenuCategory key={i} title={title} items={items} breads={breads} addToOrder={addToOrder} />
))}
```